### DISCUSSION TITLE
تعیین محدودیت متغیر در کانفیگ ها

### DISCUSSION THREAD

**Discussion by IRANWEBFA** [Gozargah/Marzban#270] [Ideas]
سلام
یه قابلیت اگر ممکنه اضافه کنید برای کانفیگ ها
مثلا من با پروتکل Vmess از طریق ای پی تمیز 100 گیگ محدودیت میزارم همچنین میخوام یه کانفیگ vless هم بسازم که روش سرور تانل کانفیگ شده و رو این vless محدودیت 30 گیگ بزارم
یعنی هر کاربر 100 گیگ vmess داشته باشه و 30 گیگ vless

**Comment by M03ED** [comment]
موردی شبیه به همین ولی به صورت ضریب مصرف متفاوت برای هر نود قبلا پیشنهاد داده شده و تا جایی که خبر دارم تو برنامه شون هست ولی زمان می‌بره تا انجام بشه
فعلا دارن همین نسخه رو دی باگ می کنن تا مرزبان نود به صورت استیبل در دسترس قرار بگیره
